<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Updateteacher</class>
 <widget class="QMainWindow" name="Updateteacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Update Teacher</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Address</string>
    </property>
    <widget class="QPlainTextEdit" name="address">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>401</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Fill all Information</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>teacherName</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DOB:</string>
     </property>
     <property name="buddy">
      <cstring>dobDate</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CNIC/Bform</string>
     </property>
     <property name="buddy">
      <cstring>cnicLineEdit</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
     <property name="buddy">
      <cstring>gender</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Qualification</string>
     </property>
     <property name="buddy">
      <cstring>qualification</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="teacherName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>301</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cnicLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>301</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qualification">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>291</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="gender">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="dobDate">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>70</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="ageSpinBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
     <property name="buddy">
      <cstring>qualification</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="department">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>291</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Designation</string>
     </property>
     <property name="buddy">
      <cstring>qualification</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="designation">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>291</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Joining Date</string>
     </property>
     <property name="buddy">
      <cstring>qualification</cstring>
     </property>
    </widget>
    <widget class="QDateEdit" name="joiningDate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
     <property name="buddy">
      <cstring>qualification</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="salary">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone:</string>
     </property>
     <property name="buddy">
      <cstring>qualification</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone_number">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>291</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>291</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
     <property name="buddy">
      <cstring>qualification</cstring>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: &quot;white&quot;;</string>
    </property>
    <property name="text">
     <string>Delete Teacher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idSearch">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Teacher ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
